docs/how-to-install.md

```diff
@@ -6,145 +6,151 @@ license: "This software is licensed under the Apache License version 2."
+### Configure the Gravitino server environment variables
+
+Gravitino exposes several environment variables through the `conf/gravitino-env.sh` file. To configure these environment variables, you can update this file.
+
+### Configure the Gravitino catalogs
+
+Gravitino supports multiple catalogs. You can configure the catalog-level configurations by updating the related configuration file in the `catalogs/<catalog-provider>/conf` directory. For example, the `catalogs/hive/conf/hive.conf` file provides configurations for the Hive catalog. The configurations you set in the catalog configuration file will apply to all the catalogs of the same type that you created. For detailed configurations about each catalog, see related catalog documentation.
 
 :::note
-Gravitino takes the catalog configurations in the following order:
+Gravitino takes the catalog configurations in the following order of precedence:
 
-1. Catalog `properties` specified in catalog creation API or REST API.
+1. Catalog `properties` specified in the catalog creation API or REST API.
 2. Catalog configurations specified in the catalog configuration file.
 
-The catalog `properties` can override the catalog configurations specified in the configurationfile.
+The catalog `properties` overrides the catalog configurations specified in the configuration file.
 :::
 
-Gravitino supports passing in catalog-specific configurations if you add `gravitino.bypass.`. For example, if you want to pass in the HMS-specific configuration `hive.metastore.client.capability.check` to the underlying Hive client in the Hive catalog, add the `gravitino.bypass.` prefix.
+Gravitino supports passing in catalog-specific configurations by adding the `gravitino.bypass.` prefix to the specific catalog configurations. For example, you can pass in the HMS-specific configuration `hive.metastore.client.capability.check` to the underlying Hive client in the Hive catalog if you add the `gravitino.bypass.` prefix to the configuration.
 
-Also, Gravitino supports loading catalog-specific configurations from external files. For example,you can put your own `hive-site.xml` file in the `catalogs/hive/conf` directory, and Gravitino loads it automatically.
+Gravitino also supports loading catalog-specific configurations from external files. You just need to put your `hive-site.xml` file in the `catalogs/hive/conf` directory, and Gravitino will load it automatically.
 
-#### Start Apache Gravitino server
+### Start the Gravitino server
 
-After configuring the Gravitino server, start the Gravitino server by running:
+- Start the Gravitino server
 
-```shell
-./bin/gravitino.sh start
-```
+    After configuring the Gravitino server, run the following command to start the Gravitino server:
 
-Alternatively, to start the Gravitino server Web UI, please run:
+    ```shell
+    ./bin/gravitino.sh start
+    ```
```

Comment:
Please ALWAYS try avoid indentation by 4 spaces.
In markdown, 4-spaces indentation might be implicitly treated as code snippets.
It is annoying.

Reply:
if using 2 spaces for indentation under an ordered list or unordered list, it looks the content is not indented properly when previewing in local, as below
<img width="1162" alt="image" src="https://github.com/user-attachments/assets/9d59bd94-a6b7-44db-92eb-152b0e80e45c">

Reply:
For ordered list, the suggested indentation would be 3 spaces, i.e.

````
1. text goes here
   and continues here

   ```shell
   curl -v ...
   ```

   other texts
1. This is the next item
   with long text paragraph...
````